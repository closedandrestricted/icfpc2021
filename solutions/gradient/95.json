{"vertices":[[24,69],[25,64],[45,66],[42,63],[24,42],[16,58],[12,60],[53,95],[25,49],[27,49],[25,56],[43,91],[27,58],[60,54],[45,135],[103,59],[65,50],[107,58],[6,124],[120,56],[48,145],[33,163],[119,60],[112,67],[69,118],[39,161],[70,105],[12,115],[14,117],[21,96],[108,42],[118,64],[71,98],[120,69],[103,39],[121,64],[21,91],[17,106],[120,73],[73,131],[47,170],[21,104],[59,179],[47,174],[86,90],[82,110],[85,125],[58,184],[88,90],[87,69],[98,66],[65,190],[95,85],[98,121],[70,190],[82,76],[96,104],[62,194],[52,203],[64,197],[99,75],[104,81],[108,118],[101,78],[111,102],[107,52],[155,135],[125,114],[104,45],[110,85],[160,133],[123,103],[114,98],[135,119],[120,66],[174,170],[169,153],[126,69],[91,50],[166,151],[86,56],[142,111],[114,183],[99,53],[120,192],[135,60],[101,41],[118,189],[118,181],[135,54],[97,47],[122,124],[105,39],[123,115],[115,13],[113,183],[104,181],[115,180],[124,169],[120,167],[113,35],[114,38],[149,46],[153,43],[118,23],[124,141],[130,159],[124,23],[130,138],[136,163],[163,39],[163,43],[173,33]]}