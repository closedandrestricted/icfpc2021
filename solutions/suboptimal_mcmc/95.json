{"vertices":[[26,68],[26,63],[46,66],[42,63],[23,43],[16,60],[12,60],[25,123],[25,49],[23,50],[25,56],[34,114],[27,58],[60,54],[45,135],[114,54],[65,50],[110,53],[40,116],[97,62],[48,145],[33,163],[96,58],[81,50],[59,116],[39,161],[62,104],[36,107],[38,105],[21,96],[113,37],[83,56],[62,98],[88,56],[115,32],[84,59],[26,94],[27,105],[88,52],[69,126],[47,170],[23,105],[60,179],[47,174],[78,90],[82,110],[82,124],[58,184],[80,91],[96,73],[98,61],[65,190],[89,92],[95,121],[70,190],[89,78],[93,102],[62,194],[52,203],[64,198],[101,69],[99,87],[104,118],[101,83],[105,110],[106,63],[145,139],[121,116],[114,63],[110,85],[151,139],[122,105],[114,98],[132,117],[118,66],[156,177],[148,161],[124,68],[111,78],[150,158],[110,85],[142,112],[120,138],[102,79],[107,135],[134,61],[110,71],[111,135],[118,135],[134,55],[96,77],[115,125],[105,72],[116,116],[104,72],[112,128],[104,125],[115,127],[112,147],[114,152],[97,69],[96,66],[149,48],[153,46],[109,65],[106,139],[106,157],[114,63],[113,140],[104,164],[161,38],[163,41],[169,30]]}