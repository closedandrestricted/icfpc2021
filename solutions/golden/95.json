{"vertices":[[51,100],[52,95],[31,94],[35,97],[65,96],[63,94],[65,98],[137,133],[72,96],[72,98],[72,90],[124,130],[75,90],[37,117],[96,166],[95,131],[31,119],[92,128],[135,180],[104,118],[95,156],[98,139],[106,122],[117,120],[103,137],[93,138],[91,129],[133,169],[136,168],[117,155],[76,128],[110,121],[84,128],[115,123],[71,126],[108,123],[113,158],[126,162],[118,121],[92,129],[97,127],[124,165],[85,120],[94,124],[72,118],[92,110],[98,140],[83,115],[72,116],[50,120],[47,131],[77,109],[62,117],[108,132],[80,106],[51,128],[101,121],[72,111],[70,124],[75,114],[40,138],[71,119],[99,131],[69,123],[90,132],[50,83],[105,82],[92,116],[59,85],[71,134],[110,85],[87,126],[82,136],[93,105],[91,118],[133,53],[120,65],[92,126],[58,70],[120,69],[54,63],[101,112],[76,107],[64,62],[72,94],[105,123],[71,73],[73,97],[78,104],[111,120],[69,67],[66,158],[69,77],[59,164],[73,85],[71,104],[64,106],[75,104],[84,115],[89,115],[69,86],[66,85],[122,113],[125,109],[81,83],[73,142],[77,124],[86,80],[66,142],[73,118],[135,104],[132,105],[142,111]]}